templates/todo/projects/csharp-cosmos-sql/README.md

```diff
@@ -42,49 +44,16 @@ The following prerequisites are required to use this application. Please ensure
 - [Node.js with npm (16.13.1+)](https://nodejs.org/) - for the Web frontend
 
 ### Quickstart
+To learn how to get started with any template, follow the steps in [this quickstart](https://learn.microsoft.com/azure/developer/azure-developer-cli/get-started?tabs=localinstall&pivots=programming-language-csharp) with this template (`Azure-Samples/todo-csharp-cosmos-sql`).
 
-The fastest way for you to get this application up and running on Azure is to use the `azd up` command. This single command will create and configure all necessary Azure resources - including access policies and roles for your account and service-to-service communication with Managed Identities.
-
-1. Open a terminal, create a new empty folder, and change into it.
-2. Run the following command to initialize the project.
+This quickstart will show you how to authenticate on Azure, initialize using a template, provision infrastructure and deploy code on Azure via the following commands:
 
-```bash
-azd init --template todo-csharp-cosmos-sql
 ```
-
-This command will clone the code to your current folder and prompt you for the following information:
-
-- `Environment Name`: This will be used as a prefix for the resource group that will be created to hold all Azure resources. This name should be unique within your Azure subscription.
-
-3. Run the following command to package a deployable copy of your application, provision the template's infrastructure to Azure and also deploy the application code to those newly provisioned resources.
-
-```bash
+azd auth login
+azd init --template Azure-Samples/todo-csharp-cosmos-sql
 azd up
 ```
```

Comment:
I see we're linking to the quickstart for the full walkthrough, and we want to present a concise summary here. If so, I wonder if it's better to format if so that the user understands this isn't a "copy-paste" type situation, rather the concise  workflow.

Something along the lines of:
```bash
# Log in to azd. Only required once per-install.
azd auth login

# First-time project setup. Initialize a project in the current directory, using this template. 
azd init --template Azure-Samples/todo-csharp-cosmos-sql

# Provision and deploy to Azure
azd up
```

The blueprint can then be modified to suit your needs. When you're ready to deploy the changes, simply run `azd up`.